variables.tf

```diff
@@ -68,6 +68,30 @@ variable "docker_version" {
   type        = "string"
 }
 
+variable "graphite_host" {
+  description = "Graphite server configured in Shunting Yard to send metrics to."
+  type        = "string"
+  default     = "localhost"
+}
+
+variable "graphite_port" {
+  description = "Graphite server port."
+  type        = "string"
+  default     = "2003"
+}
+
+variable "graphite_namespace" {
+  description = "Namespace for all metrics sent to Graphite from this Shunting Yard instance."
+  type        = "string"
+  default     = "com.hotels"
+}
+
+variable "graphite_prefix" {
+  description = "Prefix addded to all metrics sent to Graphite from this Shunting Yard instance."
+  type        = "string"
+  default     = "shuntingyard"
```

Comment:
Can we enforce that this is set to something and not have a default? Our recommendation is for people to use the "name" of the account they are running in, not sure if we can default to that?

Reply:
Variable is removed.